src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -265,19 +267,17 @@ public static void openConsole(File file) throws IOException {
         if (file == null) {
             return;
         }
-
-        String absolutePath = file.getAbsolutePath();
-        absolutePath = absolutePath.substring(0, absolutePath.lastIndexOf(File.separator) + 1);
+        String absolutePath = file.toPath().toAbsolutePath().getParent().toString();
```

Comment:
`File` is part of the old Java IO API. `Path` is its new replacement in the Java NIO (New IO) API. With file.toPath() or path.toFile() one can convert between these two APIs. The Java NIO has a lot of benefits to offer, which may be the reason why it is used here.